<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.176136 rgba(91, 156, 129, 255), stop:0.943182 rgba(135, 207, 227, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="barraBusqueda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="infoBasicaMascota">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Nombre:</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>9</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Especie:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Raza:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Color:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nombre Tutor:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Numero Telefono:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Direccion:</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="datoTablaMedica">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Alergias:</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>9</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Registro de Operaciones</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vacunas Suministradas</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="botonBuscar">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregarFicha">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>490</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Ficha</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidFichas">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="botonFichaSelected">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver Ficha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonVolver">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>650</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
